LibreHardwareMonitorLib/Hardware/Gpu/NvidiaGpu.cs

```diff
@@ -389,9 +389,11 @@ public NvidiaGpu(int adapterIndex, NvApi.NvPhysicalGpuHandle handle, NvApi.NvDis
                                         }
                                     }
 
+                                    _memoryLoad = new Sensor("GPU Memory", (_loads?.Length ?? 0) + (_powers?.Length ?? 0), SensorType.Load, this, settings);
```

Comment:
This still means that the sensor won't be available on Linux. _memoryLoad needs to be moved back to the bottom (at index 3). The other sensors need to be properly offset for this.